src/modules/movieCounter.js

```diff
@@ -0,0 +1,8 @@
+const movieCount = (movie) => {
+  let count = 0;
+  for (let i = 0; i < movie.length; i += 1) {
+    count += 1;
+  } return count;
+};
```

Comment:
- [ ] Amazing job with all the functionalities and design. However, the counter is supposed to count HTML elements on the page. You are still rendering the length of data retrieved from the API. I think you are using a div with a class name `container` for your movies, so you can use `document.querySelectorAll('.container')` to get all the HTML nodes with that class name. Kindly perform the counting on the HTML nodes. ✍️
- [ ] [OPTIONAL] I highly suggest you do the same thing for the comment counter. 😎️

Reply:
Hi, I changed the movie counter to count HTML elements on the page but for the comment counter, how can I select the dom element for a specific comment container because its id is dynamically created? Can you explain this to me?

Reply:
Hi @Rhaegar121, you can call the comments counter function after the comments are rendered on the screen. You can give the specific comment a class name to assist you in the counting process.

![Screenshot from 2023-02-09 12-06-58](https://user-images.githubusercontent.com/17243325/217767207-8d326dc0-2acd-4d64-bb07-146c0490801f.png)

Reply:
The counter could look like this.

![image](https://user-images.githubusercontent.com/17243325/217768210-5063f0ea-f167-42e3-89bf-719c90ea2792.png)

Reply:
Hi @amejid ,

Yeah I have tried the querySelectorAll method but it didn't work. The output show only 1 for all the movies' comment. I think I have to select the particular comment section but I don't know how to do or it is possible to do.

![Screenshot 2023-02-09 155203](https://user-images.githubusercontent.com/111948881/217771504-e6cba930-1a42-4510-98b0-4730d4c813cf.jpg)
![Screenshot (55)](https://user-images.githubusercontent.com/111948881/217771570-45536151-0de4-4f76-aecd-0bff1822485d.png)
![Screenshot (56)](https://user-images.githubusercontent.com/111948881/217771623-89ac7328-4682-4b74-910e-845b95d1a71c.png)

Reply:
If you have any references or examples, I'll be happy to learn from them. 😊

Reply:
I hope the issue cleared up after the call.